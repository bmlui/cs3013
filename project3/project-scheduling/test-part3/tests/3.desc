RR: CPU can be idle, i.e., more new jobs arrive after previously job(s) completed
