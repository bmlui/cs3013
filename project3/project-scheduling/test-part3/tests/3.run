./scheduler 0 RR tests/3.in 2
