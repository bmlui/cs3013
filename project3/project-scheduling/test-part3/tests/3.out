Execution trace with RR:
t=0: [Job 0] arrived at [0], ran for: [1]
t=2: [Job 1] arrived at [2], ran for: [2]
t=4: [Job 2] arrived at [3], ran for: [2]
t=6: [Job 3] arrived at [4], ran for: [2]
t=8: [Job 1] arrived at [2], ran for: [2]
t=10: [Job 2] arrived at [3], ran for: [2]
t=12: [Job 3] arrived at [4], ran for: [2]
t=14: [Job 1] arrived at [2], ran for: [1]
t=15: [Job 2] arrived at [3], ran for: [2]
t=17: [Job 2] arrived at [3], ran for: [2]
t=19: [Job 2] arrived at [3], ran for: [2]
t=21: [Job 2] arrived at [3], ran for: [1]
End of execution with RR.
